### DISCUSSION TITLE
zh-hant/chapter_data_structure/number_encoding/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1512] [Announcements]
# zh-hant/chapter_data_structure/number_encoding/

動畫圖解、一鍵執行的資料結構與演算法教程

https://www.hello-algo.com/zh-hant/chapter_data_structure/number_encoding/

**Comment by YChanHuang** [comment]
這個補碼的真的是很難理解，目前先跳過...
刷題需要費心思了解這個嗎？
